meta/src/main/proto/gravitino_meta.proto

```diff
@@ -118,13 +118,17 @@ message Group {
   AuditInfo audit_info = 5;
 }
 
+message SecurableObject {
+  string full_name = 1;
```

Comment:
Now, kv doesn't support to use ID to query name. So I use name temporarily. I will refactor here after the kv is ready.